src/SDK.ts

```diff
@@ -342,14 +371,16 @@ export default class SDK implements ISDK {
         "Get Reconnectable chat Started");
     }
 
-    const endpoint = `${this.omnichannelConfiguration.orgUrl}/${OmnichannelEndpoints.LiveChatGetReconnectableChatsPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${this.omnichannelConfiguration.orgId}?channelId=${this.omnichannelConfiguration.channelId}`;
+    const requestPath = `/${OmnichannelEndpoints.LiveChatGetReconnectableChatsPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${this.omnichannelConfiguration.orgId}?channelId=${this.omnichannelConfiguration.channelId}`;
```

Comment:
I see there's a typo on requestPath with `[orgId]/[widgetId]/[orgId]`. Can we switch it to `[orgId]/[widgetId]/[widgetId]`?

Reply:
getReconnectableChats does not have requestId. I haven't changed URL, it was as this before my changes

Reply:
Agree, but the contract of the service looks like this:

`reconnectablechats/{orgId}/{widgetAppId}/{requestId}`

So, there's a typo in the OCSDK that needs to be addressed.

Reply:
After discussion, we saw that OrgId is passed as a RequestId. It needs to be fixed at some future date, but not in scope of this task